### PR TITLE
[ASA-263][AK] fix various test cases and make build runable

### PR DESCRIPTION
**Pull request by arkowalski**

### CODE REVIEW

File: app/FrontendModule.scala [hunk 1]
```diff
@@ -51,10 +51,10 @@ class FrontendModule(val environment: Environment, val configuration: Configurat
...
 
     bindProperty("agent-guidance-page.full-url")
-    bindProperty("agent-services-account-frontend.external-url")
-    bindProperty("agent-services-account-frontend.start-path")
-    bindProperty("government-gateway-registration-frontend.external-url")
-    bindProperty("government-gateway-registration-frontend.sosRedirect-path")
+    bindProperty("microservice.services.agent-services-account-frontend.external-url")
+    bindProperty("microservice.services.agent-services-account-frontend.start-path")
+    bindProperty("microservice.services.government-gateway-registration-frontend.external-url")
+    bindProperty("microservice.services.government-gateway-registration-frontend.sosRedirect-path")
```
Comment (johnrevill) [inline app/FrontendModule.scala]: Is 'sos' still the current name of that registration journey?

Comment (arkowalski) [reply app/FrontendModule.scala]: its been copied from agent-subscription-fe

File: app/uk/gov/hmrc/agentoverseassubscriptionfrontend/controllers/auth/SubscriptionAuthAction.scala [hunk 2]
```diff
@@ -43,8 +43,8 @@ class SubscriptionAuthActionImpl @Inject()(
   val env: Environment,
   val authConnector: AuthConnector,
   val config: Configuration,
-  @Named("agent-services-account-frontend.external-url") agentServicesAccountBaseUrl: String,
-  @Named("agent-services-account-frontend.start-path") agentServicesAccountHomePath: String)
+  @Named("microservice.services.agent-services-account-frontend.external-url") agentServicesAccountBaseUrl: String,
```
Comment (johnrevill) [inline app/uk/gov/hmrc/agentoverseassubscriptionfrontend/controllers/auth/SubscriptionAuthAction.scala]: Could these two config parameters for a single URL be merged into just the one config?

Comment (arkowalski) [reply app/uk/gov/hmrc/agentoverseassubscriptionfrontend/controllers/auth/SubscriptionAuthAction.scala]: we can do that in upcoming tickets

**Pull request merged**